roles/eth2client/defaults/lighthouse/validator.yml

```diff
@@ -10,7 +10,7 @@ validator_image_name: sigp/lighthouse:latest
 validator_start_args: >
   lighthouse
   {% if testnet_type == 'custom' %}
-  --network-dir="{{validator_container_dir}}/custom_testnet_config"
+  --testnet-dir="{{validator_container_dir}}/custom_testnet_config"
```

Comment:
I believe they changed it from testnet to network with v1.0 release? Why change it back?

Reply:
I looked into it, I guess the change was the other way around? 
`Lighthouse v1.0.6-f183af2` doesn't recognize `--network-dir` but it does recognize `--testnet-dir`.

Reply:
https://github.com/sigp/lighthouse/pull/1751

Looks like they changed `--testnet` to `--network`, but not the other flag. Opening an issue so they can fix the inconsistency would be neat.

Reply:
Ahh okay, I'll create an issue to fix the inconsistency.

Reply:
I thought I had seen the `--network-dir` somewhere before, lighthouse beacon already uses ` --network-dir <DIR>
            Data directory for network keys. Defaults to network/ inside the beacon node dir.`. So I guess they left this as `--testnet-dir` to avoid confusion.